swlor2_2da/portraits.2da

```diff
@@ -4455,3 +4455,19 @@
 4451   nautF10_       1      168    ****            0      ****          
 4452   nautF11_       1      168    ****            0      ****          
 4453   nautF12_       1      168    ****            0      ****          
+4454   ewokM1_        0      169    ****            0      ****
```

Comment:
There are a bunch of portraits listed here but I don't see the equivalent tga files in the swlor2_portraits folder.